### ISSUE TITLE
Error in Chrome 67+ the permission window is never displayed

### ISSUE THREAD

**Issue by woakas** [ibm-bluemix-push-notifications/Web_HelloPush#6]
I have a issue for chrome 67+ the permission window is never displayed, according to the documentation ([push-notification](https://developers.google.com/web/fundamentals/push-notifications/subscribing-a-user)) the parameter applicationServerKey is mandatory form chrome, Is it possible to add this parameter? I think that this parameter should be added inside the backend to support it.

https://github.com/ibm-bluemix-push-notifications/Web_HelloPush/blob/f75253774071844c7ab4db6a1b7171dc9f40631d/Example_WebSite/public/BMSPushSDK.js#L343

**Comment by AnanthaKrish** [comment]
@woakas , applicationServerKey is an optional value for Chrome.  The issue you are getting might be because of the manifest file issues. Make sure you are following the docs and giving the proper values. 
We will update our SDK to add the applicationServerKey soon.

**Comment by woakas** [comment]
Hi @AnanthaKrish sorry but the notifications are working fine in chrome 66 and 65 but in the version 68 don't work. in the 67 in some navegators works and anothers not. 

In the [mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/API/PushManager/subscribe ) says that this parameter is required:
![image](https://user-images.githubusercontent.com/131361/42462076-d3b80e9a-8367-11e8-838c-2d2038a135d5.png)

In the google website [2](https://developers.google.com/web/fundamentals/push-notifications/subscribing-a-user#applicationserverkey_option) have:

_Technically, the applicationServerKey is optional. However, the easiest implementation on Chrome requires it, and other browsers may require it in the future. It's optional on Firefox._

So, why this behavior in the new versions of chrome?

**Comment by AnanthaKrish** [comment]
@woakas Yes, from Chrome 68 (beta) the `applicationServerKey ` is mandatory for subscribing push notifications. On the next release of SDK we will add `applicationServerKey ` option .

**Comment by woakas** [comment]
@AnanthaKrish Thanks, I'll wait for this.

Regards.

**Comment by woakas** [comment]
@AnanthaKrish any update on this issue?

**Comment by AnanthaKrish** [comment]
Hi @woakas , Not yet . I will soon update it and definitely will update this comment  .

**Comment by woakas** [comment]
Thank you!!

**Comment by woakas** [comment]
@AnanthaKrish Any update on this issue?

**Comment by AnanthaKrish** [comment]
@woakas Hi , `applicationServerKey` will be added in the next release. Thank you for the reminder 👍
